### PR TITLE
Master tony

### PR DESCRIPTION
**Pull request by Clase026**
!!If this bricks the program for you, tell me and we can try to get it sorted out!!

Report view now creates .docx files containing tables with all the information for games/merchandise.

Consider attempting to install python-docx before merging this branch, as it can be a bit troublesome.

[Games Report.docx](https://github.com/rowex105/sportsMerchandise/files/3163/Games.Report.docx)
[Merchandise Report.docx](https://github.com/rowex105/sportsMerchandise/files/3164/Merchandise.Report.docx)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,2 @@
+python-docx
```
Comment (rowex105) [inline requirements.txt]: Is the library called "python-docx" or just "docx"?

Comment (Clase026) [inline requirements.txt]: The library is just called docx

File: src/Views/ReportView.py [hunk 2]
```diff
@@ -3,50 +3,94 @@
 from src.Views.MainView import *
 from src.Models.DatabaseModel import *
 from src.Models.Classes import *
+from docx import *
```
Comment (Clase026) [inline src/Views/ReportView.py]: This is where python-docx is being imported, the library being docx

File: requirements.txt (hunk 1 above)
Comment (rowex105) [inline requirements.txt]: Ok so here just make it "docx" then

Comment (Clase026) [inline requirements.txt]: No can do boss. docx is something different from python-docx.

Comment (rowex105) [inline requirements.txt]: NM, there are two different packages... one is python-docx and one is docx, you're right :)

**Comment by rowex105** [comment]
Looks good to me, merge when you're ready

**Pull request merged**